sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/data/repository/RepositoryImp.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmrc.sample_compose_fragments.data.repository
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.toArgb
+import uk.gov.hmrc.components.compose.ui.theme.HmrcBlack
+import uk.gov.hmrc.components.compose.ui.theme.HmrcBlue
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGreen1
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGreen2
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGrey1
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGrey2
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGrey3
+import uk.gov.hmrc.components.compose.ui.theme.HmrcPink
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTeal
+import uk.gov.hmrc.components.compose.ui.theme.HmrcWhite
+import uk.gov.hmrc.components.compose.ui.theme.HmrcWhiteBackground
+import uk.gov.hmrc.components.compose.ui.theme.HmrcWhiteDark
+import uk.gov.hmrc.components.compose.ui.theme.HmrcYellow
+import uk.gov.hmrc.sample_compose_fragments.domain.model.ColorItem
+import uk.gov.hmrc.sample_compose_fragments.domain.repository.Repository
+import javax.inject.Inject
+
+class RepositoryImpl @Inject constructor() : Repository {
+
+    override suspend fun getLists(): List<ColorItem> {
+        val colorList = arrayListOf<ColorItem>()
+        colorList.add(ColorItem("White (${HmrcWhite.hexToString()})", HmrcWhite))
+        colorList.add(ColorItem("Black (${HmrcBlack.hexToString()})", HmrcBlack))
```

Comment:
Please can we swap these round so it matches the order in the existing sample app? Black first, then white?